bin/cross-compile-istioctl

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# bazel's go_binary cross-compilation isn't ready (see
+# github.com/bazelbuild/rules_go/issues/70). Use normal golang toolchain to
+# cross compile istioctl for windows and osx.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# build with bazel and run /bin/init.sh first so dependencies are setup
+# correctly.
+bazel build //cmd/istioctl:istioctl
+bin/init.sh
+
+# set linker flags for populating version info
+LDFLAGS=""
+while read line; do
+  read SYMBOL VALUE < <(echo $line)
+  LDFLAGS=${LDFLAGS}" -X istio.io/manager/cmd/version.${SYMBOL}=${VALUE}"
+done < <(./bin/get_workspace_status)
+
+GOOS=darwin GOARCH=amd64 go build -ldflags "${LDFLAGS}" istio.io/manager/cmd/istioctl
+mv istioctl istioctl-osx
+
+# install missing spf13/cobra dependency (windows only)
+go get github.com/inconshreveable/mousetrap
```

Comment:
Looks like the opposite case. bazel *is* observing build flags (or more accurately, golang toolchain is setting build flags correctly) and bazel is only building for linux so `windows` build flag isn't defined. Not usually a problem unless you decide to build non-linux golang binaries outside of bazel ...


```golang
$ head vendor/github.com/spf13/cobra/command_win.go
// +build windows

package cobra

import (
        "os"
        "time"

        "github.com/inconshreveable/mousetrap"
)
```